pq-crypto/EVP_pq_constants.h

```diff
@@ -0,0 +1,38 @@
+//
+// Created by lamjyoti on 8/11/2021.
+//
+
+#ifndef AWSLC_EVP_PQ_CONSTANTS_H
+#define AWSLC_EVP_PQ_CONSTANTS_H
+
+
+/* TLDR s2n: 0=true, -1=false,
+ * TLDR aws-lc: 1=true, 0=false
```

Comment:
This comment is a little misleading; it's not so much `true` or `false` as it is `success` or `failure`. In C in terms of booleans evaluation, 0 is always `false`, and any non-zero value is always `true`. These macros are more about indicating whether a function executed successfully or not.